test/service.spec.js

```diff
@@ -757,6 +790,38 @@
         return expect(didError).to.equal(true);
       });
     });
+    describe('Service date sync functions', function() { 
+      beforeEach(function(done) {
+        AWS.config.update({
+          systemClockOffset: 0
+        });
+        done();
+      });
+      it('should find clock skew if service time is skewed within 30 seconds', function() {
```

Comment:
Can you rename this test? 

> should find clock skew if service time is skewed within 30 seconds

`isClockSkewed` should be false if the service time is skewed within 30 seconds. Maybe rename it to make it clear that the clock should be considered skewed if service time is not within 30 seconds of the client time.